src/Turn.js

```diff
@@ -15,12 +16,22 @@ class Turn {
   }
 
   evaluateGuess() {
-    if(this.guess === this.card.correctAnswer) {
+    if (this.guess === this.card.correctAnswer) {
+      this.guessValue = true;
       return true
     } else {
+      this.guessValue = false;
```

Comment:
I'm not sure this line of code is necessary since in the giveFeedback() method below, I only specify if the this.guessValue === true...do something.